### DISCUSSION TITLE
Support for DateOnly and TimeOnly

### DISCUSSION THREAD

**Discussion by kevingolding2001** [MoonStorm/FastCrud#196] [Ideas]
This is a fantastic library that has made my code so much simpler by utilizing generics ie. (InsertAsync<T>(record);)

However it seems it does not work with some DotNet and Postgresql types.

I've tried using DateOnly and TimeOnly in my C# code, and using the Postgresql equivalents of date and time in the database, but I'm getting null values passed through when trying to insert a new record.

What would be required to add support for DateOnly and TimeOnly? 

Comments elsewhere suggest that the actual mapping is not done in this library, but it relies on Dapper for this.  I've already confirmed that these types work in the latest pre-release version of Dapper if I explicitly create the sql statement. 

Does this mean it is as simple as adding something to the _simpleSqlTypes type array?

If there is more to it than that, what is it and how can I contribute?

**Comment by MoonStorm** [comment]
We'll take a look.

**Comment by MoonStorm** [comment]
We've added support on our side for these .NET6 types, updated the dependencies for the main lib and tests to the latest versions and released a new version on NuGet. However, we still found that we needed to add overrides in case of several providers.

You might not need them in your case but, if you do and for the time being, [here they are](https://github.com/MoonStorm/FastCrud/wiki/Property-value-mappings#dateonly-and-timeonly).
